process/OpenHW-Specification-Process.md

```diff
@@ -0,0 +1,180 @@
+| 5			| Contribution Process 						| Follows Eclipse Development Process (EDP) / Committers merging pull requests	/ Contributors sign Eclipse Contributor Agreement (ECA) or Member Committer and Contributor Agreement (MCCA)		| 			 
+
+
+#### Publication of the Specification
+
+| Number 	| Issue 									| Comment                                               							|
+| :---		| :------ 									| :-------------------------------------------------------------------------------- |
+| 1 		| Specification published by				| OpenHW Group on Github and/or ReadtheDocs and/or website								|
+| 2 		| Copyright holder on specification			| OpenHW Group on behalf of contributors (members), who hold the copyright jointly. 		|
+| 3			| License used for publication 				| Same license as used for contributions 											|
+| 4			| Copyright Grant included in specification publication 	 | As per publication license:										|	
+| 5.1		| Apache 2.0								| Yes, copyright grant included with OpenHW Group publication 							|
+| 5.2		| Solderpad 2.1								| Yes, copyright grant included with OpenHW Group publication 							|
+| 5.3		| Creative Commons Attribution Share Alike 4.0 International license | Yes, copyright grant included with OpenHW Group publication 			|								
+| 6			| Patent Grant included in specification publication 	|	As per publication license											|
+| 6.1		| Apache 2.0								| No patent grant provided by OpenHW but patents rights granted under contribution	|
+| 6.2		| Solderpad 2.1								| No patent grant provided by OpenHW but patents rights granted under contribution	|
+| 6.3		| Commons Attribution Share Alike 4.0 International license | No patent grant included with OpenHW publication					|								
+| 7			| Essential patents disclosed by OpenHW		| No																				|
+| 8 		| License text included within the specification text |  No - referral only 													|
+| 9			| "No-warranty" or disclaimer				| Required																|
+
+
+### Specification Revision Numbers
+
+OpenHW Group Specifications shall use semantic versioning https://semver.org/ with the Revision in the form X.Y.Z.
+The Revision number is combined with the Specification State (below), for example "1.0.0 - Released" to indicate both the Revision number and State.
+
+
+
+### Specification States
+
+The ratification or completion State is meant to be written together with the Title and Revision in each OpenHW Group specification so that a reader will be aware of the state of completion of the Specification.
+These are the OpenHW Group Specification States:
+
+- “OpenHW Group Specification - In Development"
+- "OpenHW Group Specification - In Review" (optional)
+- "OpenHW Group Specification - Release Candidate"
+- "OpenHW Group Specification - Released"
+
+#### "OpenHW Group Specification - In Development"
+
+When the specification is initiated, all drafts will be entitled  "OpenHW Group Specification: Title : VX.Y.Z : In Development"
```

Comment:
Usually doesn't make sense to have the version in the title. Reason is that the templates print the version on the title page/header. So including it in the title would lead to having the revision number twice in most places